gloo/docs/user_guides/external_api_routing.md

```diff
@@ -1,113 +1,158 @@
 ---
 title: External API Routing
 weight: 30
+description: Example of routing to an external upstream.
 ---
 
 ## External API Routing
 
-In this tutorial, we'll take a look at routing to services that live outside of your deployment platform and that have not been configured for automatic discovery. You can consider that these could be existing monoliths or static endpoints that do not lend themselves easily to be discovered. Gloo's power comes from it's ability to live in unpredictable and dynamic environments just fine, but for those use cases where we need to explicitly add upstreams, we can do that following these steps.
-
+In this tutorial, we'll take a look at routing to services that live outside of your deployment platform and that have
+not been configured for automatic discovery. You can consider that these could be existing monoliths or static endpoints
+that do not lend themselves easily to be discovered. Gloo's power comes from it's ability to live in unpredictable and
+dynamic environments just fine, but for those use cases where we need to explicitly add upstreams, we can do that
+following these steps.
```

Comment:
I've been manually wrapping at 120 characters as otherwise its hard to read on these crazy big monitors we've got in office without shrinking IDE. Not sure best convention.